src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -133,6 +137,26 @@ void LiveFunctionsDataView::UpdateSelectedFunctionId() {
 void LiveFunctionsDataView::OnSelect(const std::vector<int>& rows) {
   UpdateHighlightedFunctionId(rows);
   UpdateSelectedFunctionId();
+
+  if (!rows.empty()) {
+    const FunctionInfo& function = *GetFunctionInfoFromRow(rows[0]);
+    std::string function_name = orbit_client_data::function_utils::GetDisplayName(function);
+
+    uint64_t function_id = GetInstrumentedFunctionId(rows[0]);
+    std::vector<const orbit_client_protos::TimerInfo*> timers =
+        app_->GetAllTimersForHookedFunction(function_id);
+    std::vector<uint64_t> histogram_data;
```

Comment:
Nit: As C++'s `map` (aka transform) looks horrible, I would consider renaming that vector to make it immediately obvious that this is the durations of the timers/functions calls.

E.g. `function_durations`/`timer_durations`.